<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Review</class>
 <widget class="QDialog" name="Review">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>白词斩</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>复习</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_question_English">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_judge">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DCHYJianWeiDGB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>加油！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
